Opacity/OpacitySwiftWrapper.swift

```diff
@@ -236,4 +236,17 @@ public class OpacitySwiftWrapper {
       }
     }
   }
+
+  // github
+  public static func getGithubProfile() async throws -> (json: String, proof: String) {
+    return try await withCheckedThrowingContinuation { continuation in
+      OpacityObjCWrapper.getGithubProfile { (json, proof, error) in
+        if let error {
+          continuation.resume(throwing: error)
```

Comment:
Did you test this on the example add? Just add the code on the view controller and let's always add it to the repo example to make sure we can run it